### PR TITLE
🐛 modify pom to resolve vulnerabilities

### PR DESCRIPTION
**Pull request by GeorgeLuo**
Aggregates the dependency upgrades.
Will not pass validator-java tests.

### CODE REVIEW

File: validator/java/pom.xml [hunk 1]
```diff
@@ -19,16 +19,16 @@
         <validator.proto.directory>${project.basedir}/src/main/proto</validator.proto.directory>
 
         <!-- library versions -->
-        <build-helper-maven-plugin.version>3.0.0</build-helper-maven-plugin.version>
+      <build-helper-maven-plugin.version>3.1.0</build-helper-maven-plugin.version>
```
Comment (rsimha) [inline validator/java/pom.xml:22]: Seems like the indentation is broken here and everywhere else. Can you fix?

Comment (GeorgeLuo) [reply validator/java/pom.xml:22]: yeah, I can do that, this will obfuscate the material changes, but now that you've seen them it's not a worry

Comment (rsimha) [reply validator/java/pom.xml]: It's possible to [hide whitespaces from the diffs](https://github.com/ampproject/amphtml/pull/27951/files?w=1) by adding `?w=1` to the Github URL. Still looks good!

**Comment by rsimha** [comment]
The Travis failures in the java validator are known, and are in the process of being fixed. Meanwhile, I'm force-merging this to eliminate all security vulnerabilities due to old dependency versions in `validator/java/pom.xml`.

**Pull request merged**